mantidproject/mantid
# CorrectFlightPaths fix and rename

CorrectFlightPaths had a bug after the recent histogram changes. This meant that the spectra in the output workspace were empty. This PR fixes that, and adds a unit test.

The algorithm is also renamed to ConvertToConstantL2 and the documentation updated to state that this is not intended as part of any reduction process for IN5.

**To test:**

Check that the file `UnitTest/ILL/IN5/104007.nxs` can be loaded and run via ConvertToConstantL2. Afterwards looking at the detector table the detectors should all have the same distance to the sample.

Running via CorrectFlightPaths should give the same results but the deprecated warning.

Documentation should show CorrectFlightPaths is deprecated, and documentation should be updated for ConvertToConstantL2.

Fixes #17873.

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
